app/presenters/unsubscribe_link_presenter.rb

```diff
@@ -17,7 +17,11 @@ def call
 private
 
   def url
-    PublicUrlService.url_for(base_path: "/email/unsubscribe/#{uuid}?title=#{escaped_title}")
+    PublicUrlService.url_for(base_path: base_path)
+  end
+
+  def base_path
+    "/email/unsubscribe/#{uuid}?title=#{escaped_title}"
```

Comment:
I'd dare suggest that having a method for this adds unnecessary indirection. I'd maybe suggest we did url as:

```
escaped_title = ERB::Util.url_encode(title)
base_path =  "/email/unsubscribe/#{uuid}?title=#{escaped_title}"
PublicUrlService.url_for(base_path: base_path)
```

and ditched the other two methods